content/en/resources/defend-your-moat.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
A test example that will be updated before linking to the resources page in the footer.